code/script.js

```diff
@@ -1,14 +1,30 @@
-// All the DOM selectors stored as short variables
-const chat = document.getElementById('chat')
+// DOM selectors stored as variables
+const chat = document.getElementById('chat'),
+      startButton = document.getElementById('start-btn'),
+      form = document.getElementById('name-form'),
+      inputWrapper = document.getElementById('input-wrapper'),
+      inputValue = document.getElementById('name-input');
 
-// Global variables, if you need any, declared here
+// Functions
 
-// Functions declared here
+// This function adds a chat bubble in the correct place based on who the sender is
 
-// This function will add a chat bubble in the correct place based on who the sender is
 const showMessage = (message, sender) => {
   if (sender === 'user') {
-    chat.innerHTML += `
+
+    // Isabel's console.log (just as a test)
+
+    console.log(message)
```

Comment:
For next time, maybe remove the console.log for the final push

Reply:
Oh right! It's easy to miss these things.